src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -41,6 +98,23 @@ public class ScriptTagHelper : TagHelper
         public string FallbackSrc { get; set; }
 
         /// <summary>
+        /// A comma separated list of globbed file patterns of JavaScript scripts to fallback to in the case the
+        /// primary one fails (as specified in the src attribute).
```

Comment:
nit: primary one isn't necessarily specified in the `src` attribute.  might be `asp-src-include` and perhaps `asp-src-exclude`